onnxruntime/test/contrib_ops/gemm_fastgelu_op_test.cc

```diff
@@ -38,8 +38,15 @@ static void RunGemmFastGeluGpuTest(const std::vector<float>& input_data, const s
   }
 
   std::vector<std::unique_ptr<IExecutionProvider>> execution_providers;
-  execution_providers.push_back(DefaultRocmExecutionProvider());
-  tester.Run(OpTester::ExpectResult::kExpectSuccess, "", {}, nullptr, &execution_providers);
+
+  execution_providers.emplace_back(DefaultRocmExecutionProvider(/*test_tunable_op=*/true));
```

Comment:
Can you use somthing similar to this https://github.com/microsoft/onnxruntime/blob/769febb2d974653cecb65eebb4f10a786d436327/onnxruntime/test/providers/cpu/math/gemm_test.cc#L17-L23

This allows other EP to enable TunableOp test more easily in the future. Also, in the future, this will allow us to change how we construct the EP in a more centralized way.

Reply:
Done.